paddle/fluid/pybind/pybind.cc

```diff
@@ -1031,6 +1032,19 @@ All parameter, weight, gradient are variables in Paddle.
     }
     return ret_values;
   });
+  m.def("get_op_attrs_default_value",
+        [](py::bytes byte_name) -> paddle::framework::AttributeMap {
```

Comment:
Why not use `const std::string op_type` as input?

Reply:
Both of `const std::string` and `py::bytes` can be use.